shipyard.md

```diff
@@ -165,49 +206,54 @@ metadata:
   name: shipyard-sockshop
 spec:
   stages:
-  - name: dev
-    sequences:
-    - name: artifact-delivery
-      tasks:
-      - name: deployment
-        properties:  
-          deploymentstrategy: direct
-      - name: test
-        properties:
-          teststrategy: functional
-      - name: evaluation 
-      - name: release 
-
-  - name: hardening
-    sequences:
-    - name: artifact-delivery
-      triggers:
-      - dev.artifact-delivery.finished
-      tasks:
-      - name: deployment
-        properties: 
-          deploymentstrategy: blue_green_service
-      - name: test
-        properties:  
-          teststrategy: performance
-      - name: evaluation
-      - name: release
-        
-  - name: production
-    sequences:
-    - name: artifact-delivery 
-      triggers:
-      - hardening.artifact-delivery.finished
-      tasks:
-      - name: deployment
-        properties:
-          deploymentstrategy: blue_green
-      - name: release
-      
-    - name: remediation
-      tasks:
-      - name: remediation
-      - name: evaluation
+    - name: dev
+      sequences:
+        - name: artifact-delivery
```

Comment:
In the example, can you please change the sequence name `artifact-delivery` to just `delivery`.